substrate-ingest/src/sink.ts

```diff
@@ -393,6 +395,10 @@ export class PostgresSink implements Sink {
             throw e
         }
     }
+
+    private isAcalaEvmExecuted(event: Event) {
+        return 'contract' in event.args && 'logs' in event.args
```

Comment:
there are more than two fields but i believe it's enough
https://acala.subscan.io/extrinsic/1896016-2?event=1896016-45